locales/en.yml

```diff
@@ -187,3 +187,8 @@ en:
         debug: |+
           Most verbose logging. This causes 'debug'
           level logs to be emitted.
+        force_shutdown: |+
+          Force logstash to exit during shutdown even
+          if there are still inflight events in memory.
+          By default, logstash will refuse to quit until all
+          received events have been pushed to the outputs.
```

Comment:
I like this. If this should cause data loss (dead letter replay isn't available yet, right?), can you add a note indicating the data loss here?